tests/cheetah/remove_call_data/test_can_remove_calls/multiline.tmpl

```diff
@@ -1,7 +1,5 @@
         foo  bar
-        foo $dotted.name bar
+        foo ${dotted.name} bar
         foo $cheetahvar bar
         foo $cheetah.var bar
-        foo $(
-            x.upper() for x in mylist
-        ) bar
```

Comment:
this is the output -- the input is still multilined.  I made the argument thingy more clever with non-important whitespace